serveradmin/serverdb/models.py

```diff
@@ -49,7 +49,7 @@
 
 HOSTNAME_VALIDATORS = [
     RegexValidator(
-        r'\A(\*\.)?([a-z0-9]+[\.\-])*[a-z0-9]+\Z', 'Invalid hostname'
+        r'\A(\*\.)?([a-z0-9]+([\.]|[\-]*))*[a-z0-9]+\Z', 'Invalid hostname'
```

Comment:
I think we can remove a few square brackets and we don't need to escape the hyphen.
```suggestion
        r'\A(\*\.)?([a-z0-9]+(\.|-+))*[a-z0-9]+\Z', 'Invalid hostname'
```

Reply:
True, I will adjust the migration script.

Reply:
Good suggestion, @meikomeik. Especially the `+` after the hyphen was important to avoid ReDOS attacks.